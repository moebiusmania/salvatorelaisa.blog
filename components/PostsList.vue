<template>
  <ul>
    <slot></slot>
  </ul>
</template>

<style scoped>
ul {
  border-top: 1px solid rgb(229, 231, 235);
  margin: var(--sp-4) 0;
  padding: 0;
}
</style>