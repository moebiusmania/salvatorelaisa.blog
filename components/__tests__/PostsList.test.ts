import { describe, it, expect } from "vitest";
import { mount } from "@vue/test-utils";
import PostsList from "../PostsList.vue";

describe("PostsList", () => {
  // Snapshot test
  it("renders correctly", () => {
    const wrapper = mount(PostsList, {
      slots: {
        default: "<li>Test post item</li>",
      },
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
