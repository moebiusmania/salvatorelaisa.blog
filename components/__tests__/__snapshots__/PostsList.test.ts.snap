// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`PostsList > renders correctly 1`] = `
"<ul data-v-6c67b3d0="">
  <li>Test post item</li>
</ul>"
`;
